cartoframes/data/observatory/enrichment/enrichment_service.py

```diff
@@ -290,7 +290,7 @@ def _build_polygons_query_variable_with_aggregation(self, variable, aggregation)
                 """.format(
                     column=variable.column_name,
                     geo_column=self.geojson_column,
-                    aggregation=variable_agg)
+                    aggregation=variable_agg.lower())
```

Comment:
I am seeing a problem related to that. The most important, I am doing:
```
if variable_agg == 'SUM'
``` 
but, what happens if the user does `aggr = 'sum'`? It will fail (or the result will be wrong).

I am going to add more changes here, doing the `lower` from the beginning and changing the 'SUM' validation

Reply:
Why? That's only happens in the format of the query.